pkg/access/commands/use.go

```diff
@@ -78,6 +82,35 @@ func AccessDetails() *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.StringVarP(&outputFormat, outputFlagName, "o", "cli", "output format")
+	flags.BoolVarP(&run, runFlagName, "r", false, "output format")
 
 	return cmd
 }
+
+func verifyOutputFormatIsSupported(ctx context.Context, client *aponoapi.AponoClient, id string, outputFormat string) error {
+	session, _, err := client.ClientAPI.AccessSessionsAPI.GetAccessSession(ctx, id).Execute()
+	if err != nil {
+		return fmt.Errorf("access session with id %s not found", id)
+	}
+
+	for _, supportedFormat := range session.ConnectionMethods {
+		if supportedFormat == outputFormat {
+			return nil
+		}
+	}
+
+	return fmt.Errorf("unsupported output format: %s. use one of: %s", outputFormat, strings.Join(session.ConnectionMethods, ", "))
+}
+
+func executeAccessDetails(ctx context.Context, accessID string, accessDetails *clientapi.AccessSessionDetailsClientModel) error {
+	if accessDetails.GetCli() == "" {
+		return fmt.Errorf("--run flag is not supported for access id %s", accessID)
+	}
+
+	err := exec.CommandContext(ctx, "sh", "-c", accessDetails.GetCli()).Run() //nolint:gosec // This is a command that should be executed for the user
+	if err != nil {
+		return fmt.Errorf("error executing command:\n%s\n%w", accessDetails.GetCli(), err)
+	}
```

Comment:
does this execute the command in the current shell?
I must be using the same shell the cli is run from to inherit all the variables, params and credentials and set everything properly for the user